lib/open_api_spex/validation.ex

```diff
@@ -0,0 +1,10 @@
+defmodule OpenApiSpex.Validation do
+  @moduledoc """
+  The result of a cast or validation
+  """
+
+  defstruct schema: nil,
+            schemas: [],
+            value: nil,
+            error: nil
```

Comment:
Can this be an error list? Even if initially it is always a single error for `Cast` validations.

Reply:
By the end of it, this will be `errors: list()`. Right now, I'm just doing the minimal first step.